sql/ballots/ballots_test.go

```diff
@@ -123,36 +138,48 @@ func TestLatest(t *testing.T) {
 	require.NoError(t, err)
 	require.Equal(t, types.LayerID(0), latest)
 
-	ballot := types.NewExistingBallot(types.BallotID{1}, types.EmptyEdSignature, types.EmptyNodeID, types.LayerID(11))
+	ballot := types.NewExistingBallot(
+		types.BallotID{1},
+		types.EmptyEdSignature,
+		types.EmptyNodeID,
+		types.LayerID(11),
+	)
 	require.NoError(t, Add(db, &ballot))
 	latest, err = LatestLayer(db)
 	require.NoError(t, err)
 	require.Equal(t, ballot.Layer, latest)
 
-	newBallot := types.NewExistingBallot(types.BallotID{2}, types.EmptyEdSignature, types.EmptyNodeID, types.LayerID(12))
+	newBallot := types.NewExistingBallot(
+		types.BallotID{2},
+		types.EmptyEdSignature,
+		types.EmptyNodeID,
+		types.LayerID(12),
+	)
 	require.NoError(t, Add(db, &newBallot))
 	latest, err = LatestLayer(db)
 	require.NoError(t, err)
 	require.Equal(t, newBallot.Layer, latest)
 }
 
-func TestLayerBallotBySmesher(t *testing.T) {
+func TestLayerBallotByAtxID(t *testing.T) {
 	db := sql.InMemory()
 	lid := types.LayerID(1)
-	nodeID1 := types.RandomNodeID()
-	nodeID2 := types.RandomNodeID()
+	nodeID1 := types.NodeID{1}
+	nodeID2 := types.NodeID{2}
 	ballots := []types.Ballot{
 		types.NewExistingBallot(types.BallotID{2}, types.EmptyEdSignature, nodeID1, lid.Add(1)),
 		types.NewExistingBallot(types.BallotID{3}, types.EmptyEdSignature, nodeID2, lid),
 	}
+	ballots[0].AtxID = types.ATXID{1}
+	ballots[1].AtxID = types.ATXID{2}
 	for _, ballot := range ballots {
 		require.NoError(t, Add(db, &ballot))
 	}
 
-	prev, err := LayerBallotByNodeID(db, lid, nodeID1)
+	prev, err := LayerBallotByAtxID(db, lid, ballots[0].AtxID)
 	require.ErrorIs(t, err, sql.ErrNotFound)
 	require.Nil(t, prev)
-	prev, err = LayerBallotByNodeID(db, lid, nodeID2)
+	prev, err = LayerBallotByAtxID(db, lid, ballots[1].AtxID)
```

Comment:
Why is the variable named "prev" instead of "ballot"?

Reply:
like previous ballot, but yes ballot is clearer